web/packages/teleport/src/Integrations/status/AwsOidc/AwsOidcDashboard.tsx

```diff
@@ -18,18 +18,43 @@
 
 import React from 'react';
 
+import { Flex, H2, Indicator } from 'design';
+
+import { Danger } from 'design/Alert';
+
 import { AwsOidcHeader } from 'teleport/Integrations/status/AwsOidc/AwsOidcHeader';
 import { useAwsOidcStatus } from 'teleport/Integrations/status/AwsOidc/useAwsOidcStatus';
 import { FeatureBox } from 'teleport/components/Layout';
+import {
+  AwsResource,
+  StatCard,
+} from 'teleport/Integrations/status/AwsOidc/StatCard';
 
-// todo (michellescripts) after routing, ensure this view can be sticky
 export function AwsOidcDashboard() {
-  const { attempt } = useAwsOidcStatus();
+  const { statsAttempt, integrationAttempt } = useAwsOidcStatus();
+
+  if (statsAttempt.status == 'processing') {
+    return <Indicator />;
+  }
+  if (statsAttempt.status == 'error') {
+    return <Danger>{statsAttempt.statusText}</Danger>;
+  }
+  if (!statsAttempt.data) {
+    return;
```

Comment:
```suggestion
    return null;
```

`undefined` isn't a valid `ReactNode`